Branch-SDK/src/main/java/io/branch/referral/util/BranchEvent.java

```diff
@@ -244,16 +244,48 @@ public String getEventName() {
      * Logs this BranchEvent to Branch for tracking and analytics
      *
      * @param context Current context
-     * @return {@code true} if the event is logged to Branch
+     * @return {@code true}
      */
     public boolean logEvent(Context context) {
-        boolean isReqQueued = false;
+        logEvent(context, null);
+        return true;
```

Comment:
Technically this is changing some behavior. 

Before we would return the fact it got queued. I'd recommend keeping it and returning the result of `logEvent(context, null);` instead of that method being void, return the `boolean` if it got queued as before in `logEvent(Context context)`